website/content/docs/concepts/transform.mdx

```diff
@@ -8,30 +8,127 @@ description: >-
 
```

Comment:
Description could use some tightening up re extraneous 'of'; suggestion:

"Part of Vault's Advanced Data Protection solution, protecting sensitive information
  stored outside Vault."

Reply:
Updated